client/state/experiments/reducer.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External Dependencies
+ */
+import { findIndex, tail } from 'lodash';
+import { Action } from 'redux';
+
+/**
+ * Internal Dependencies
+ */
+import { EXPERIMENT_FETCH, EXPERIMENT_ASSIGN, CURRENT_USER_RECEIVE } from 'state/action-types';
+import { ExperimentState, ExperimentAssign } from 'state/experiments/types';
+
+/**
+ * Attempt to get the anon id for the user, if set
+ */
+function getAnonIdFromCookie(): string | null {
+	if ( document && document.cookie != null ) {
+		const cookieString = document.cookie;
```

Comment:
Poking around the code, I see that the analytics module uses `cookie.parse()` to get `tk_ai`. Any reason not to do something similar here?

https://github.com/Automattic/wp-calypso/blob/bcddb6c2e3e9c983e7dccb8acaff5da784ad2b65/client/lib/analytics/index.js#L119-L121

There's also a function to get the anon ID:

https://github.com/Automattic/wp-calypso/blob/bcddb6c2e3e9c983e7dccb8acaff5da784ad2b65/client/lib/analytics/ad-tracking.js#L1464-L1472

Reply:
Nice find! I'll change this. I also looked but started by looking at `package.json` and didn't see us explicitly depending on any useful cookie package, so I stopped there.